src/commands/trigger-workflows-for-all-modified.yml

```diff
@@ -176,6 +176,8 @@ steps:
           custom_params=`cat /tmp/swissknife/workflow_params.json`
           echo $custom_params
 
+          if [ "$CI_PULL_REQUEST" != "" ]; then CIRCLE_BRANCH=$CIRCLE_BRANCH/head; fi
```

Comment:
this seems to be a deprecated env var. https://circleci.com/docs/2.0/env-vars/

what if we did this

```
if [[ "$CIRCLE_PR_NUMBER" != "" ]]; then
  CIRCLE_BRANCH="pull/$CIRCLE_PR_NUMBER";
fi
```

I am still testing if this will work. will confirm in ~10

Reply:
that ^ seems to work.

you can test with `beta.swissknife.dev` :)

Reply:
@roopakv :1st_place_medal: 

I've also added this to `trigger-pipeline-if-modified` in b00a220948b6f818ed390176272fb8dafde26f4b since both of these areas can trigger the pipeline from a forked PR.